.github/workflows/autobuild.yml

```diff
@@ -80,42 +73,40 @@ jobs:
              draft:                 false
 
 
-         ### CANCEL ### can be used for development concerning release-creation
-         #- name:                    Cancelthrougherroe
-         #  run:                     myundefinedfunction
-
-
   release_assets:
```

Comment:
Start the actual build process.

Reply:
Hm, what would be the added value of such a comment? It's just a more elaborate re-phrasing of the `name:` below to me.